antiope-dynamodb/src/Antiope/DynamoDB/Types.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+module Antiope.DynamoDB.Types where
+
+import Data.String           (IsString)
+import Data.Text             (Text)
+import GHC.Generics
+import Network.AWS.Data.Text (FromText (..), ToText (..))
+
+newtype TableName = TableName
+  { text :: Text
```

Comment:
can we make it something more descriptive, like `unTableName` or `tableName`?
Do we even need it? `toText` gives you exactly this value anyway. I think we should treat this type as a value by itself, not trying to extract its internals.

Reply:
Perhaps we can drop the field altogether.